core-plugins/src/test/java/co/cask/hydrator/plugin/batch/BatchPluginsTestSuite.java

```diff
@@ -25,7 +26,6 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-  BatchCubeSinkTestRun.class,
```

Comment:
yeah, i wanted it to be on the same package as the source class, so can call getProperties method for testing. so moved it.